trivy-image-scan/action.yml

```diff
@@ -59,7 +59,7 @@ runs:
         TRIVY_OUTPUT=""
         if [ "${{ inputs.output-mode }}" = "github" ]; then
           TRIVY_FORMAT="sarif"
-          TRIVY_OUTPUT="''trivy-results.sarif''"
+          TRIVY_OUTPUT="trivy-results.sarif"
```

Comment:
I can't recall why this had the weird quoting but it seems to be breaking it now and there were no notes in the original PR. Verified it works with this fix in https://github.com/hypertrace/config-service/pull/285